spanner/spannertest/integration_test.go

```diff
@@ -1109,6 +1109,13 @@ func TestIntegration_ReadsAndQueries(t *testing.T) {
 				{int64(4), nil, "p"},
 			},
 		},
+		{
```

Comment:
nit: Could we also add a LEFT JOIN test here? I've verified that it does work, but adding a test case could prevent potential regressions in the future.